app/src/app/components/contents/manageAccess/utils/manageRolesColumns.tsx

```diff
@@ -24,44 +24,46 @@ export const roleDefaultColumns = (scrollAreaHeightClass = "h-14") => [
       const users = getValue();
 
       return (
-        <ScrollArea className={scrollAreaHeightClass} type="auto">
-          <div className="flex flex-wrap gap-1 italic text-xs pl-0.5">
-            {users?.length === 0
-              ? "None"
-              : users.map((user, index) => (
-                  <div key={user.id}>{index === users.length - 1 ? user.username : user.username + ", "}</div>
-                ))}
-          </div>
-        </ScrollArea>
+        <div
+          className={cn(
+            "flex flex-wrap gap-1 italic text-xs pl-0.5 overflow-auto [&::-webkit-scrollbar]:w-2",
+            rowMaxHeight
+          )}
+        >
+          {users?.length === 0
+            ? "None"
+            : users.map((user, index) => (
+                <div key={user.id}>{index === users.length - 1 ? user.username : user.username + ", "}</div>
+              ))}
```

Comment:
do we need them in separate keyed divs? If not then this would be easier, and can also be applied to `setupManageRolesColumns` here, and (in manageUsersColumns.tsx) `columnHelper.accessor("roles"`

```suggestion
          {users?.length === 0
            ? "None"
            : users.map((u) => u.username).join(", ")}
```